### DISCUSSION TITLE
Bitbucket list of black list

### DISCUSSION THREAD

**Discussion by geovanepires** [actions/virtual-environments#6368] [General questions]
Hi guys,

I'm having problems with the phantomjs installation scripts, the file is on bitbucket.org and this site is blacklisted and that's why my security systems are blocking the download.

Is anyone having this same problem? Do we have an alternative?

Thank you 

![Screen Shot 2022-10-07 at 9 59 29 AM](https://user-images.githubusercontent.com/50925098/194559590-4f57bad1-5cea-4c97-b780-a776596e3c82.png)

**Comment by al-cheb** [comment]
Hey @geovanepires.
What about this https://github.com/Medium/phantomjs/releases/download/v2.1.1/phantomjs-2.1.1-linux-x86_64.tar.bz2 source?

**Reply by geovanepires** [reply]
Hi @al-cheb 

Thanks for your answare, but in the phantomjs.sh script the url is pointing to Bitbucket.

I'm going to make a pull request to fix this info in the repo, does it make sense?
